pkg/consensus/consensus.go

```diff
@@ -328,7 +331,24 @@ func (s *service) getBlock(h util.Uint256) block.Block {
 
 func (s *service) getVerifiedTx(count int) []block.Transaction {
 	pool := s.Config.Chain.GetMemPool()
-	txx := pool.GetVerifiedTransactions()
+
+	var (
+		has int
+		txx []*transaction.Transaction
+	)
+
+	if s.dbft.ViewNumber > 0 {
```

Comment:
We can initialize txx with capacity of `s.lastProposal` to avoid reallocations on `append`.